README.md

```diff
@@ -97,6 +98,25 @@ tabs:
   meta: tabs/meta/page
 ```
 
+Altering the default fields is also possible (like any Kirby Blueprint) :
```

Comment:
@Daandelange  I would prefer to have this section removed, as the plugin is still in an early state and this might still be in flux. I’d rather wait until the backend has fully stabilized before adding a recommendation for extending blueprints. Maybe we’ll find a better way in the future like providing a hook that accepts an array of field. Some of the current options like sitemap priority will probably get a switch as a config option in the future, since they are not that relevant to most users.

Reply:
Ok, I'll push a new commit to correct.
(I think the sitemap priority already is an switchable option)